lib/src/DOtherSide.cpp

```diff
@@ -1546,6 +1547,66 @@ void dos_qsettings_delete(DosQSettings* vptr)
 }
 #pragma endregion
 
+#pragma region QTimer
+
+DosQTimer *dos_qtimer_create()
+{
+    return new QTimer();
+}
+
+void dos_qtimer_delete(DosQTimer *vptr)
+{
+    auto qobject = static_cast<QObject*>(vptr);
+    if(qobject)
+        qobject->deleteLater();
+}
+
+void dos_qtimer_set_interval(DosQTimer *vptr, int interval)
+{
+    auto timer = static_cast<QTimer*>(vptr);
+    if (timer) {
+        timer->setInterval(interval);
+    }
+}
+
+int dos_qtimer_interval(DosQTimer *vptr)
+{
+    auto timer = static_cast<QTimer*>(vptr);
+    return timer ? timer->interval() : 0;
```

Comment:
Should return imho `-1`; `0` is the default (valid) value

Reply:
Fixed